### PR TITLE
FluentD 1.0

### PR DESCRIPTION
**Pull request by arcivanov**
Support multi-bulk chunks
Support custom-sized bulks
Support varying default retry sleep time
Do not lose nanoseconds in format
Log records that are too large into the error log
Fix unit tests

### CODE REVIEW

**Comment by arcivanov** [comment]
@roiravhon Please review and merge

**Comment by roiravhon** [comment]
Thank you @arcivanov!
We will get to it soon, and release after testings

**Comment by arcivanov** [comment]
@roiravhon any news on this?

**Comment by roiravhon** [comment]
Not yet @arcivanov, we need to thoroughly test this change as this plugin is widely used.
i'll keep you posted

**Comment by arcivanov** [comment]
FYI, We have been running this in production on two clusters of 3 aggregators each for a week now with no observable issues.

**Comment by roiravhon** [comment]
Good to hear!
Our platform validation team is actively working on making sure everything is perfect. 
Stay tuned

File: README.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ Logz.io plugin for [Fluentd](http://www.fluentd.org)
 =============
 With fluent-plugin-logzio you will be able to use [Logz.io](http://logz.io) as output the logs you collect with Fluentd.
 
+## Requirements
+
+| fluent-plugin-may_awesome | Fluentd     | Ruby   |
```
Comment (roiravhon) [inline README.md]: may_awesome ?

**Review by roiravhon — APPROVED** [review]
Thanks @arcivanov!
We are good to merge. Sorry it took a while, took us a bit of time to get the tests going.

Just one comment here, and we will merge and distribute it

File: README.md (hunk 1 above)
Comment (arcivanov) [reply README.md]: No idea, will fix it :D

**Comment by arcivanov** [comment]
@roiravhon I have expanded and fixed some README issues and added additional documentation. Please review.

**Comment by roiravhon** [comment]
@arcivanov thanks a lot for this.
Pushed to rubygems, and sorry for the merge delay!

**Pull request merged**